vllm/worker/model_runner.py

```diff
@@ -125,13 +125,7 @@ def _prepare_prompt(
             prompt_len = len(prompt_tokens)
             prompt_lens.append(prompt_len)
             prefix_len = 0
-            prefix = seq_group_metadata.prefix
-            if prefix is not None and prefix.computed:
-                prefix_len = prefix.get_length()
-                prompt_tokens = prompt_tokens[prefix_len:]
-                prefix_block_tables.append(prefix.get_block_numbers())
-            else:
-                prefix_block_tables.append([])
+            prefix_block_tables.append([])
```

Comment:
@SageMoore Was thinking more about this. Is there a separate spot in the code where we propagate the information about which prefixes where found in the hash table?

With manual prefix caching case, the prefix is passed by the user, and we set the `context_lens` (i.e. the length of the prefix) based on this. This metadata is then used by the model to only run forward on the new input_tokens and to use the kv_caches of the cached prefix during attention (e.g. calling `context_forward_attn` https://github.com/vllm-project/vllm/blob/063d2fb9394823be4428094dfa80ff1df18b0826/vllm/model_executor/layers/attention.py#L165)

I wasn't sure if there is a separate spot in the code where this logic sits

Reply:
This should be addressed now